commands/api.go

```diff
@@ -1010,15 +1013,22 @@ func getAccessToken() (string, error) {
 	var token string = "DUMMY TOKEN"
 	var err error
 
-	props, err := ReadProps(Properties.PropsFile)
-	if err == nil {
-		if len(props["APIGW_ACCESS_TOKEN"]) > 0 {
-			token = props["APIGW_ACCESS_TOKEN"]
-		}
+	// If the api gw access token override has been set, use it instead of the default
+	whisk.Debug(whisk.DbgInfo, "api gw access token override is '%s'\n", ApiGwAccessToken)
```

Comment:
@mdeuser, might not want to print the access token if it is sensitive information.

Reply:
i just checked the `bx` cli, and id does not display even these "short lived" tokens..